src/Sarif/Writers/ResultLogJsonWriter.cs

```diff
@@ -261,6 +262,42 @@ public void WriteInvocation(Invocation invocation)
 
             _jsonWriter.WritePropertyName("invocation");
             _serializer.Serialize(_jsonWriter, invocation, typeof(Invocation));
+
+            _writeConditions |= Conditions.InvocationWritten;
```

Comment:
oops.

---
In reply to: [61611090](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61611090) [](ancestors = 61611090)